test/e2e/kubeproxy.go

```diff
@@ -494,11 +518,6 @@ func (config *KubeProxyTestConfig) createPod(pod *api.Pod) *api.Pod {
 	if err != nil {
```

Comment:
why is this an if statement, vs `Expect(err).NotTo(HaveOccurred())`? just wondering..